internal/store/mutatingwebhookconfiguration.go

```diff
@@ -31,7 +31,6 @@ import (
 )
 
 var (
-	descMutatingWebhookConfigurationHelp          = "Kubernetes labels converted to Prometheus labels."
```

Comment:
We did not have this and a few other labels, but since we now filter out labels by default if users want we can add them for this and other resources like ConfigMaps for example.

Reply:
Agreed :+1:, it would prevent this feature from being half-baked.